cairo/tests/utils/strategies.py

```diff
@@ -174,6 +174,7 @@ def non_default_strategy(default):
                 non_default_strategy(default),
                 min_size=min_size,
                 max_size=15,
+                dict_class=defaultdict,
```

Comment:
what's the default_factory for this default_dict ? does it need to be specified?

Reply:
good point